### ISSUE TITLE
gt_segmentations missing

### ISSUE THREAD

**Issue by Rahul-Narasimhan** [MIC-DKFZ/nnUNet#2298]
I ran a 2d nnunet on a custom dataset of ~1300 CT scans. Training cases were ~1100 CT and Validation was ~200 CT. Input format is all .nii.gz format.
I ran nnUNetv2_plan_and_preprocess -d <dataset> --verify_dataset_integrity overnight. I was not sure if the command ran fully, but I was able to see the fingerpring and plans json files. 

I started the training using nnUNetv2_train <dataset> 2d 0 -device cuda --c . The training ran for 1000 epochs and even the predictions for validation cases were generated. However after that I ran into this error.
FileNotFoundError: [Errno 2] No such file or directory: '/nnUNet/nnUNet_preprocessed/<Dataset_name>/gt_segmentations'.

I feel the plan and preprocess did not complete properly. 
I did not want to re run the command again as i was not sure if it would over write any existing data. 

Could you please help with the issue ? 
Thanks in Advance.

**Comment by JackRio** [comment]
You could just run the command again and tag it with a different dataset ID to not change the existing dataset. I would recommend storing logs when you leave stuff overnight. Or you could integrate WandB with the nnUNet trainer. It is straightforward to set up and you can then verify if your command ran successfully to completion.

**Comment by saikat-roy** [comment]
Can you verify if the folder `/nnUNet/nnUNet_preprocessed/<Dataset_name>/gt_segmentations` actually exists for your dataset? I think it's failing while calculating the accuracy statistics after the prediction because it's not finding a ground truth folder.

**Comment by Rahul-Narasimhan** [comment]
No, it does not exist. All the other data required for the training was available and even the training for 1000 epochs finished. So, I guess the preprocess command failed (because of some issue from my end) . 
Which is why I was wondering what would happen if I re run nnUNetv2_plan_and_preprocess on the same dataset. will it just create things that were missing or will it start from the beginning ?

**Comment by saikat-roy** [comment]
Sorry for the late reply. Ideally this would've been created during preprocessing. Have you managed to solve this issue?

**Comment by Rahul-Narasimhan** [comment]
Hi @saikat-roy yes, This issue is solved. However There is the issue of the background workers. I have raised another issue for that here at Issue 2310.  https://github.com/MIC-DKFZ/nnUNet/issues/2310
